app/components/avatar-list.tsx

```diff
@@ -2,9 +2,10 @@ import { take, slice } from "lodash"
 
 export type AvatarListProps = {
   list?: { name: string; initials: string; pictureUrl?: string }[]
+  toggleUserSlider: Function
```

Comment:
I think we can rename this to `onToggleUserSlider` or just `onClick` since from a consumer perspective this will read something like `<AvatarList list=[] onClick={} />`. The component shouldn't care about toggling or any other action the caller wants to implement, I think.

Regarding passing the remaining users, I think that's cool because it's a piece of information the consumer of this component doesn't have. Still, I think we should show all users (visible and hidden) in the right slider, but as a concept I think it makes a lot of sense to send the remaining.

I think the correct Typescript definition for the function will be something similar to this:

```ts
export type AvatarProps = { name: string; initials: string; pictureUrl?: string }

export type AvatarListProps = {
   list?: AvatarProps[]
   toggleUserSlider: (remaining: AvatarProps[]) => void
} & JSX.IntrinsicElements["div"]
```

Reply:
I think that's reasonable, there's only a little detail with that: I can't assign exactly `onClick` because it may cause conflicts with the react MouseEvent, so I'll go with `handleOnClick` which transmits the same perspective.
![image](https://user-images.githubusercontent.com/40028772/105491742-9125fb00-5c95-11eb-9cd2-e73eb666bd3f.png)

Regarding the remaining users, of course I could pass all the users, just as you said I assumed that we wanted to show the remaining ones, but let's go with that!

Lastly, I realised that I was passing the list to the toggleSider method mistakenly, I already fixed it (it'll go within the next commit) but the thing is that function won't have any parameters.